hooks/customization/tk-multi-publish2/collector.py

```diff
@@ -0,0 +1,153 @@
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Autodesk, Inc.
+import os
+import sgtk
+
+HookBaseClass = sgtk.get_hook_baseclass()
+
+
+class BasicSceneCollector(HookBaseClass):
+    """
+    A basic collector that handles files and general objects.
+
+    This collector hook is used to collect individual files that are browsed or
+    dragged and dropped into the Publish2 UI. It can also be subclassed by other
+    collectors responsible for creating items for a file to be published such as
+    the current Maya session file.
+
+    This plugin centralizes the logic for collecting a file, including
+    determining how to display the file for publishing (based on the file
+    extension).
+
+    In addition to creating an item to publish, this hook will set the following
+    properties on the item::
+
+        path - The path to the file to publish. This could be a path
+            representing a sequence of files (including a frame specifier).
+
+        sequence_paths - If the item represents a collection of files, the
+            plugin will populate this property with a list of files matching
+            "path".
+
+    """
+
+    @property
+    def common_file_info(self):
+        """
+        A dictionary of file type info that allows the basic collector to
+        identify common production file types and associate them with a display
+        name, item type, and config icon.
+
+        The dictionary returned is of the form::
+
+            {
+                <Publish Type>: {
+                    "extensions": [<ext>, <ext>, ...],
+                    "icon": <icon path>,
+                    "item_type": <item type>
+                },
+                <Publish Type>: {
+                    "extensions": [<ext>, <ext>, ...],
+                    "icon": <icon path>,
+                    "item_type": <item type>
+                },
+                ...
+            }
+
+        See the collector source to see the default values returned.
+
+        Subclasses can override this property, get the default values via
+        ``super``, then update the dictionary as necessary by
+        adding/removing/modifying values.
+        """
+
+        # inherit the settings from the base publish plugin
+        base_file_info = super(BasicSceneCollector, self).common_file_info or {}
+
+        icons_folders = [os.path.join(self.disk_location, "icons")]
+
+        automotive_file_info = {
+            "Wref File": {
+                "extensions": ["wref"],
+                "icon": self._get_icon_path("alias.png"),
+                "item_type": "file.wref",
+            },
+            "Catpart File": {
```

Comment:
hey @staceyoue and @rob-aitchison , you have a mismatch in your publish file type declaration

for example, here you've defined a "Catpart File" type, but in the collector you call it "Catpart Translation"
I think you copied/pasted Renault's code but in their case, they were using the "old" type (aka Catpart Translation) and not the default one (aka Catpart File)

same for all the translation type